examples/epstein_civil_violence/epstein_civil_violence/agent.py

```diff
@@ -118,7 +113,9 @@ def update_estimated_arrest_probability(self):
             ):
                 actives_in_vision += 1
         self.arrest_probability = 1 - math.exp(
-            -1 * self.model.arrest_prob_constant * (cops_in_vision / actives_in_vision)
+            -1
```

Comment:
These 3 lines probably failed Black check.

Reply:
Hmm, black format on save, you're correct. Just checked and Black did reformat those lines. How do I update this on a pull?

Reply:
Just add, commit and push again it should update automatically

Reply:
Done